circuits/circomkit.json

```diff
@@ -0,0 +1,17 @@
+{
+  "protocol": "groth16",
+  "prime": "bn128",
+  "version": "2.1.5",
+  "circuits": "./circom/circuits.json",
+  "dirPtau": "./ptau",
+  "dirCircuits": "./circom",
+  "dirInputs": "./inputs",
+  "dirBuild": "./build",
+  "optimization": 2,
+  "inspect": true,
+  "include": ["./node_modules/circomlib/circuits", "./node_modules/@zk-kit/circuits/circom"],
```

Comment:
So this is what allows us to import templates without specifying /node_modules/? i.e. https://github.com/privacy-scaling-explorations/maci/pull/1010#r1451601984

Reply:
If so, what happens if circomlib & zk-kit have a template of the same name? How do we know which we're using?

Reply:
zk-kit uses circomlib too, so it would never have a template with the same name. Same goes for every other circom project in existence I would think 🤔

Reply:
but yea if we used 2 libraries with the same template file name that would be a problem, though I don't think is one we will ever have